docs/tempo/website/operations/_index.md

```diff
@@ -13,4 +13,4 @@ Resources for deploying and running Tempo:
 - [Caching](caching)
 - [Tempo CLI](tempo_cli)
 - [Polling/Tenant Index](polling)
-- [Backend Search](backend_search)
\ No newline at end of file
+- [Backend search](backend_search)
```

Comment:
It you seen title casing, change it to sentence casing to conform to our style.

Reply:
I agree it does look odd for one to be cased differently. I think we should proceed with "Backend Search" and in the future submit a PR that moves them all to sentence casing if that's what we want.

Reply:
I'm following our style guidelines.  We use sentence case for titles and headers. I choose not to PR in technical debt, 
 which requires a future PR to change to what it can be done correctly now. I also choose not to cause "feature creep" by needing to correct everything in this documentation section in this PR; I'd never get anything finished.